src/components/hand-controls.js

```diff
@@ -244,9 +253,11 @@ module.exports.Component = registerComponent('hand-controls', {
     if (animationActive) { mesh.play(animationActive, 0); }
 
     // play new animation.
-    mesh.mixer.clipAction(animation).loop = 2200;
-    mesh.mixer.clipAction(animation).clampWhenFinished = true;
-    mesh.mixer.clipAction(animation).timeScale = timeScale;
+    clipAction = mesh.mixer.clipAction(animation);
+    if (!clipAction) { return; }
```

Comment:
different perspective, the change makes it more forgiving for users to take as example and replace with their own model -- instead of "why does the controller no longer move or react" it becomes "why do my animations not work" as it should be.  But I do understand your perspective.  Would you prefer the early-return that masks errors to be commented and disabled?